app/src/main/java/com/sdsmdg/demoexample/MainActivity.java

```diff
@@ -20,8 +20,7 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     public void showSuccessToast(View view) {
-        TastyToast.makeText(getApplicationContext(), "Download Successful !", TastyToast.LENGTH_LONG,
-                TastyToast.SUCCESS);
+        TastyToast.show(getApplicationContext(), "Download Successful !");
```

Comment:
Sorry, what? 

The red lines (starting with -) are lines that have been removed. The green lines (starting with +) are added in.